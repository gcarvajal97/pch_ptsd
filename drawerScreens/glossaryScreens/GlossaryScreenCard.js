import React, { Component } from "react";
import { Text } from "react-native";
import { Card } from "react-native-elements";
import { LOCALE } from "../../mainAndLists/MainScreen";
import i18n from "i18n-js";

export default class GlossaryScreenCard extends Component {
  render() {
    return (
      <Card
        featuredTitle={i18n.t("glossary")}
        featuredTitleStyle={{ fontSize: 40 }}
        image={require("../../assets/GlossaryScreen.jpg")}
        accessible
        accessibilityLabel={i18n.t("accessabilityLabel")}
      >
        <Text
          style={{ marginVertical: -8, fontWeight: "500", textAlign: "center" }}
        >
          {i18n.t("description")}
        </Text>
      </Card>
    );
  }
}
