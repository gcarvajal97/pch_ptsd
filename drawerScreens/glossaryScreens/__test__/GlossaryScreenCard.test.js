import React from 'react';
import GlossaryScreenCard from '../GlossaryScreenCard';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<GlossaryScreenCard />).toJSON();
    expect(tree).toMatchSnapshot();
})