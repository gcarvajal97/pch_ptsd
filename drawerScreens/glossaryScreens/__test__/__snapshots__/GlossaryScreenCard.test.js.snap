// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityLabel="[missing \\"en.glossary.accessibilityLabel\\" translation]"
  accessible={true}
  style={
    Object {
      "backgroundColor": "white",
      "borderColor": "#e1e8ee",
      "borderWidth": 1,
      "margin": 15,
      "marginBottom": 0,
      "padding": 0,
      "shadowColor": "rgba(0,0,0, .2)",
      "shadowOffset": Object {
        "height": 0,
        "width": 0,
      },
      "shadowOpacity": 1,
      "shadowRadius": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
    <View>
      <View
        accessibilityIgnoresInvertColors={true}
        style={
          Object {
            "backgroundColor": "transparent",
            "position": "relative",
          }
        }
      >
        <Image
          onLoad={[Function]}
          source={1}
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "height": undefined,
                "width": undefined,
              },
            ]
          }
          testID="RNE__Image"
          theme={
            Object {
              "colors": Object {
                "disabled": "hsl(208, 8%, 90%)",
                "divider": "#bcbbc1",
                "error": "#ff190c",
                "grey0": "#393e42",
                "grey1": "#43484d",
                "grey2": "#5e6977",
                "grey3": "#86939e",
                "grey4": "#bdc6cf",
                "grey5": "#e1e8ee",
                "greyOutline": "#bbb",
                "platform": Object {
                  "android": Object {
                    "error": "#f44336",
                    "primary": "#2196f3",
                    "secondary": "#9C27B0",
                    "success": "#4caf50",
                    "warning": "#ffeb3b",
                  },
                  "ios": Object {
                    "error": "#ff3b30",
                    "primary": "#007aff",
                    "secondary": "#5856d6",
                    "success": "#4cd964",
                    "warning": "#ffcc00",
                  },
                },
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#8F0CE8",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            }
          }
        />
        <View
          accessibilityElementsHidden={true}
          importantForAccessibility="no-hide-descendants"
          pointerEvents="none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#bdbdbd",
                "height": 150,
                "justifyContent": "center",
                "width": null,
              }
            }
            testID="RNE__Image__placeholder"
          />
        </View>
        <View
          style={
            Array [
              Object {
                "height": 150,
                "width": null,
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "stretch",
                "backgroundColor": "rgba(0, 0, 0, 0.2)",
                "bottom": 0,
                "flex": 1,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 40,
                  "fontWeight": "800",
                  "marginBottom": 8,
                }
              }
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
            >
              Glossary
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Object {
              "fontWeight": "500",
              "marginVertical": -8,
              "textAlign": "center",
            }
          }
        >
          Find word definitions by category.
        </Text>
      </View>
    </View>
  </View>
</View>
`;
